src/components/auto-complete/auto-complete.st.css

```diff
@@ -0,0 +1,37 @@
+.root {
+    width: auto;
+    font-size: 16px;
+    position: relative;
+}
+
+.auto-complete-input {
+    border: 1px solid #adbac5;
+    font-family: ArialMT;
+    font-size: 16px;
+    height: 36px;
+    width: 100%;
+    padding-left: 5px;
+}
+
+.auto-complete-list {
+    -st-extends: List;
+    background: #FFFFFF;
+}
+
+.auto-complete-input:hover {
+    cursor: pointer;
```

Comment:
Please consult @giladgrush if there should be really a "pointer" over the input. methinks it should be "caret".

Reply:
it shouldn't. this doesn't belong here